config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -9,6 +9,7 @@ periodics:
     preset-azure-cred-only: "true"
     preset-azure-anonymous-pull: "true"
     preset-azure-capz-sa-cred: "true"
+    preset-azure-cred-wi: "true"
```

Comment:
Do we want to add an entirely new preset or can we add workload identity id to one of the existing presets? (`preset-azure-cred-only` or `preset-azure-capz-sa-cred`)

cc @jsturtevant

Reply:
Let's keep it separate, I think that will make it easier to cut over to the community subscription once we are ready with that

Reply:
A new preset `preset-azure-cred-wi` was created.